SampleApp/SampleApp.Android/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System.Reflection;
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+using Android.App;
+
+// General Information about an assembly is controlled through the following 
+// set of attributes. Change these attribute values to modify the information
+// associated with an assembly.
+[assembly: AssemblyTitle("SampleApp.Android")]
+[assembly: AssemblyDescription("")]
+[assembly: AssemblyConfiguration("")]
+[assembly: AssemblyCompany("")]
+[assembly: AssemblyProduct("SampleApp.Android")]
+[assembly: AssemblyCopyright("Copyright ©  2014")]
```

Comment:
Is this file auto-generated? If not, Update the `AssemblyDescription`, `AssemblyCompany` and `AssemblyCopyright` fields as appropriate. You might look at our previous sample apps for ideas. Hopefully those were updated!

Reply:
This file is generated once I created the project. Can't find `AssemblyDescription` `AssemblyCompany ` and `AssemblyCopyright ` in the previous sample app. I update the dile should be ok now :)